addon/components/au-toaster.hbs

```diff
@@ -1,17 +1,45 @@
 {{#if this.toaster.toasts.length}}
   <div class="au-c-toaster {{this.position}}">
     {{#each this.toaster.toasts as |toast|}}
-      <AuAlert
-        @title={{toast.title}}
-        @skin={{toast.options.type}}
-        @icon={{toast.options.icon}}
-        @size="small"
-        @closable={{toast.options.closable}}
-      >
-        {{#if toast.message}}
-          <p>{{toast.message}}</p>
-        {{/if}}
-      </AuAlert>
+      {{#if toast.options.action}}
+        <AuAlert
+          @title={{toast.title}}
+          @skin={{toast.options.type}}
+          @icon={{toast.options.icon}}
+          @size="small"
+          @closable={{toast.options.closable}}
+          @isAToast={{true}}
+        >
+          <:default>
+            {{#if toast.message}}
+              <p>{{toast.message}}</p>
+            {{/if}}
+          </:default>
+          <:action>
+            <AuButton
+              @skin="link"
```

Comment:
FYI: I've locked the skin here to `link` as I don't think the other skins match the toaster context or would steal to much visual focus.